src/SFA.DAS.Tools.Support.Infrastructure/Services/EmployerCommitmentsService.cs

```diff
@@ -56,5 +60,33 @@ public async Task<StopApprenticeshipResult> StopApprenticeship(long employerAcco
                 };
             }
         }
+
+        public async Task<ApprenticeshipSummaryResult> GetApprenticeshipSummary(long apprenticeshipId, long employerAccountId)
+        {
+            try
+            {
+                if (employerAccountId <= 0)
+                {
+                    throw new ArgumentException("employerAccountId must be greater than 0", "employerAccountId");
+                }
+
+                if (apprenticeshipId <= 0)
+                {
+                    throw new ArgumentException("apprenticeshipId must be greater than 0", "apprenticeshipId");
+                }
+
+                var result = await _employerCommitmentApi.GetEmployerApprenticeship(employerAccountId, apprenticeshipId);
```

Comment:
We need to handle Apprenticeship not found scenario here and perhaps show that message to the user on the input page.

Reply:
I've left this one as is. The underlying commitments api client executes an EnsureStatusCode line which throws a generic exception which we can't query anymore. The api returns a 404 which is shown on the page to indicate it wasn't found But more can be done when we upgrade to V2